pyproject.toml

```diff
@@ -113,6 +117,8 @@ module = [
   'SOAPify.*',
   'numba.*',
   'onion_clustering.*',
-  'py.*'
+  'py.*',
+  'scipy.*'
 ]
+
 ignore_missing_imports = true
```

Comment:
Can you add an empty line at the end of the file.

Reply:
i actually can't... I really don't know why but also the current version is without the blank line.

Reply:
Ok, ignore